crypto/pki/pki.go

```diff
@@ -299,6 +347,12 @@ func (p *PKI) tellPKI() {
 	ui.PrintSelected("Root fingerprint", p.rootFingerprint)
 	ui.PrintSelected("Intermediate certificate", p.intermediate)
 	ui.PrintSelected("Intermediate private key", p.intermediateKey)
+	if p.enableSSH {
+		ui.PrintSelected("SSH user certificate", p.sshUserCert)
```

Comment:
consider saying `SSH user CA ...  & SSH host CA ...`. Otherwise it may not be clear that these are for use by the CA (for signing).